saige_assoc.py

```diff
@@ -313,6 +316,7 @@ def create_second_job(vcf_path: str) -> hb.batch.job.Job:
 @click.option(
     '--vre-files-prefix', default=dataset_path('saige-qtl/input_files/genotypes')
 )
+@click.option('--writeout-file-prefix', default=dataset_path('saige-qtl'))
```

Comment:
I just had a thought... you probably want the default location for log files to be in `-analysis`, so that you won't have any issues reading them in future